i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-centreon-sql-metrics.md

```diff
@@ -52,85 +50,80 @@ Voici le tableau des services pour ce connecteur, détaillant les métriques rat
+| *host*#centreon.dsm.host.queue.cache.count | count |
 
 </TabItem>
 <TabItem value="Execution-Time" label="Execution-Time">
 
 | Métrique                        | Unité |
 |:--------------------------------|:------|
 | services.execution.exceed.count | count |
+| *services*#list                 | N/A   |
 
 </TabItem>
 <TabItem value="Notifications-Count" label="Notifications-Count">
 
-| Métrique                               | Unité |
-|:---------------------------------------|:------|
-| notifications.sent.count               |       |
-| *poller_name*#notifications.sent.count |       |
+| Métrique                 | Unité |
+|:-------------------------|:------|
+| notifications.sent.count | count |
+| notifications.sent.count | count |
 
 </TabItem>
 <TabItem value="Poller-Delay" label="Poller-Delay">
 
-| Métrique                                    | Unité |
-|:--------------------------------------------|:------|
-| *poller_name*#centreon.poller.delay.seconds | s     |
+| Métrique                               | Unité |
+|:---------------------------------------|:------|
+| *poller*#centreon.poller.delay.seconds | s     |
 
 </TabItem>
 <TabItem value="Problems-Count" label="Problems-Count">
 
-| Métrique                              | Unité |
-|:--------------------------------------|:------|
-| total.outage.count                    |       |
-| hosts.outage.count                    |       |
-| services.outage.count                 |       |
-| *poller_name*#hosts.down.count        |       |
-| *poller_name*#services.warning.count  |       |
-| *poller_name*#services.critical.count |       |
-| *poller_name*#services.unknown.count  |       |
+| Métrique              | Unité |
+|:----------------------|:------|
+| total.outage.count    | count |
+| hosts.outage.count    | count |
+| services.outage.count | count |
 
 </TabItem>
 <TabItem value="Resources-Count" label="Resources-Count">
 
-| Métrique                                       | Unité |
-|:-----------------------------------------------|:------|
-| *poller_name*#centreon.hosts.host.count        |       |
-| *poller_name*#centreon.hosts.total.count       |       |
-| *poller_name*#centreon.hosts.up.count          |       |
-| *poller_name*#centreon.hosts.down.count        |       |
-| *poller_name*#centreon.hosts.unreachable.count |       |
-| *poller_name*#centreon.hosts.pending.count     |       |
-| *poller_name*#centreon.services.service.count  |       |
-| *poller_name*#centreon.services.total.count    |       |
-| *poller_name*#centreon.services.ok.count       |       |
-| *poller_name*#centreon.services.warning.count  |       |
-| *poller_name*#centreon.services.critical.count |       |
-| *poller_name*#centreon.services.unknown.count  |       |
-| *poller_name*#centreon.services.pending.count  |       |
+| Métrique                                   | Unité |
+|:-------------------------------------------|:------|
+| *pollers*~centreon.hosts.count             | count |
```

Comment:
Isn't "poller_name" a better name than "pollers"? Is it that we define the option for all pollers at once?